homeassistant/components/whois/sensor.py

```diff
@@ -121,6 +128,13 @@ def _ensure_timezone(timestamp: datetime | None) -> datetime | None:
         entity_registry_enabled_default=False,
         value_fn=lambda domain: getattr(domain, "reseller", None),
     ),
+    WhoisSensorEntityDescription(
+        key="status",
+        translation_key="status",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
+        value_fn=lambda domain: getattr(domain, "status", None),
```

Comment:
Should we translate the statusses as well?

Reply:
The status is a string that's returned from the whois provider, and it can be [quite diverse](https://www.icann.org/resources/pages/epp-status-codes-2014-06-16-en). So I my initial idea was to not translate them.

Reply:
I think the general better pratice is to guard for what can be expected and provide translations for. This enabled the ability to automate with it. A random text is hard to work with.

Reply:
Okay I implemented the translations. It's a bit weird, since different registars return different values, e.g.:
```
Domain Status: clientDeleteProhibited https://icann.org/epp#clientDeleteProhibited
```
So it's looking if one of the statuscode is present in the returned sting, and if that's the case it returns the snake_case version of that.